http/data-url.json

```diff
@@ -50,22 +50,19 @@
             "chrome_android": "mirror",
             "edge": {
               "version_added": "12",
-              "notes": "The maximum size supported is 4GB"
+              "notes": "Before Edge 79, the maximum size supported was 4GB."
```

Comment:
@ddbeck recommends a different style, current tense and not past tense.

Reply:
I've updated the notes according to the proposed style, however I'd like to point out it hasn't been made into a guideline.

I also happened to notice that these notes were duplicated on both the parent and a few children.  I've moved these notes to the parent feature and removed them from the children for better placement/consistency.